modules/logging-loki-zone-fail-recovery.adoc

```diff
@@ -0,0 +1,83 @@
+
+In {product-title} a zone failure happens when specific availability zone resources become inaccessible. Availability zones are isolated areas within a cloud provider's data center, aimed at enhancing redundancy and fault tolerance. If your {product-title} cluster isn't configured to handle this, a zone failure can lead to service or data loss.
+
+Loki pods are part of a link:https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/[StatefulSet], and they come with Persistent Volume Claims (PVCs) provisioned by a `StorageClass`. Each Loki pod and its PVCs reside in the same zone. When a zone failure occurs in a cluster, the StatefulSet controller will automatically attempt to recover the affected pods in the failed zone.
+
+[WARNING]
+====
+The following procedure will delete the PVCs in the failed zone, and all data contained therein.  To avoid complete data loss the replication factor field of the `LokiStack` CR should always be set to a value greater than 1 to ensure that Loki is replicating.
+====
+
+.Prerequisites
+* Logging version 5.8 or later.
+* Verify your `LokiStack` CR has a replication factor greater than 1.
+* Zone failure detected by the control plane, and nodes in the failed zone are marked by cloud provider integration.
+
+While the StatefulSet controller will automatically attempt to reschedule pods in a failed zone, since the associated PVCs are also in the failed zone, automatic rescheduling to a different zone will not work. PVCs in the failed zone must be manually deleted to allow successful recreation of the stateful Loki Pod and its provisioned PVC in the new zone.
+
+
+.Procedure
+. List the pods in `Pending` status:
+
+[source,terminal]
+----
+oc get pods --field-selector status.phase==Pending -n openshift-logging
+----
++
+.Example `oc get pods` output
+[source,terminal]
+----
+NAME                           READY   STATUS    RESTARTS   AGE # <1>
+logging-loki-index-gateway-1   0/1     Pending   0          17m
+logging-loki-ingester-1        0/1     Pending   0          16m
+logging-loki-ruler-1           0/1     Pending   0          16m
+----
+<1> These pods are in `Pending` status because their corresponding PVCs are in the failed zone.
+
+.. List the PVCs in `Pending` status:
+[source,terminal]
+----
+oc get pvc -o=json -n openshift-logging | jq '.items[] | select(.status.phase == "Pending") | .metadata.name' -r
+----
++
+.Example `oc get pvc` output
+[source,terminal]
+----
+storage-logging-loki-index-gateway-1
+storage-logging-loki-ingester-1
+wal-logging-loki-ingester-1
+storage-logging-loki-ruler-1
+wal-logging-loki-ruler-1
+----
+
+.. Delete the PVC(s) for a pod:
+[source,terminal]
+----
+oc delete pvc __<pvc_name>__  -n openshift-logging
+----
+
+.. Then delete the pod(s):
+[source,terminal]
+----
+oc delete pod __<pod_name>__  -n openshift-logging
+----
+
+Once these objects have been successfully deleted, they should automatically be recreated in an available zone.
```

Comment:
| they should automatically be recreated in an available zone.

Should we stat ewhat process/feature does the re-creating, to make it really clear? As in: _ {preoduct-title} automatically re-creates the objects in an available zone._